mamboms/setup.py

```diff
@@ -40,7 +40,7 @@
         'pyparsing==1.5.6',
         'python-memcached==1.58',
         'wsgiref==0.1.2',
-        'South==1.0.2',
+        #'South==1.0.2',
```

Comment:
Why is south commented out?

Reply:
It was giving me an error and I found the following related to south: https://docs.djangoproject.com/en/1.7/topics/migrations/#upgrading-from-south
But I have uncommented and rebuilt it seems fine now.

Reply:
See also: http://south.aeracode.org/